chapter2/tests/t9/chapter2-l4-t9-get-tcp-listening-connections-in-terminator.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# The solution should always output in the open terminator console even if you close and reopen the terminator window.
+set -eou pipefail
+
+trap 'echo ERROR!; echo on line: $LINENO in $0; cat $0 | sed "$LINENO!d"' ERR
+trap '' SIGHUP
```

Comment:
why do you need the trap?

Reply:
I use it to avoid closing the script execution if we close the terminal (parent process exit).

Reply:
use nohup
https://www.tomshardware.com/how-to/run-nohup-in-linux

Reply:
Removed the line, updated the description that script should be run with `nohup` command.